File: If you can read this.js
const toNato = (words) => {
    const alphabet = 'abcdefghijklmnopqrstuvwxyz'
    let result = []
    const arr = words.split(" ");
    for(let s1 of arr){
      for(let s2 of s1){
        if(alphabet.includes(s2.toLowerCase())){
          result.push(NATO[s2.toUpperCase()])
        }else{
          result.push(s2)
        }
      }
    }
    return result.join(" ")
}

File: If you can read this/If you can read this.swift
func toNato(_ words: String) -> String {
  var result: [String] = []
  let arr = words.split(separator: " ")
  for s1 in arr {
      for s2 in s1 {
          let key = Character(String(s2).uppercased())
          result.append(NATO[key] ?? String(s2))
      }
  }
  return result.joined(separator: " ")
}
